lighthouse-core/gather/gatherers/script-elements.js

```diff
@@ -29,7 +29,7 @@ function collectAllScriptElements() {
       id: script.id || null,
       async: script.async,
       defer: script.defer,
-      source: /** @type {'head'|'body'} */ (script.closest('head') ? 'head' : 'body'),
```

Comment:
this cast does nothing because the untyped `...getNodeDetails()` below is spreading `...any` which just spreads into the whole object being `any` :)